mappings/net/minecraft/server/network/ServerPlayerEntity.mapping

```diff
@@ -89,7 +93,9 @@ CLASS net/minecraft/class_3222 net/minecraft/server/network/ServerPlayerEntity
 	METHOD method_14252 setExperienceLevel (I)V
 		ARG 1 level
 	METHOD method_14253 getRecipeBook ()Lnet/minecraft/class_3441;
-	METHOD method_14254 sendChatMessage (Lnet/minecraft/class_2561;Lnet/minecraft/class_2556;)V
+	METHOD method_14254 sendGameMessage (Lnet/minecraft/class_2561;Lnet/minecraft/class_2556;)V
```

Comment:
Why? This is just a more specific overload of `ServerPlayerEntity.sendMessage`. Maybe you should instead rename that to `sendChatMessage`?

Reply:
I see the linked issue now, but it's wrong. All these types of messages still appear in chat. The MessageType parameter is to allow players to turn off certain types of chat messages in the game options.

Reply:
It's also the location of the message. Game info message type (bed failures, build limit, etc) appears over the hotbar, hence "game message" instead of "chat message".

Reply:
Fair enough, but this message can send messages of all 3 types right? That makes "game message" overly specific. How about:
- `sendMessage` -> `sendChatMessage`
- `sendChatMessage` -> `sendMessage`

Reply:
I don't think the `MessageType` one can send the actionbar messages.